src/TestSuite/Fixture/FixtureHelper.php

```diff
@@ -0,0 +1,161 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         5.0.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\TestSuite\Fixture;
+
+use Cake\Core\Configure;
+use Cake\Database\Driver\Postgres;
+use Cake\Datasource\ConnectionInterface;
+use Cake\Datasource\ConnectionManager;
+use Closure;
+use UnexpectedValueException;
+
+/**
+ * Helper for managing fixtures.
+ */
```

Comment:
Should this be `@internal`?

Reply:
I think it's supportable. If others want some combination strategy, they can use it.

Reply:
Maybe we don't want o support the transaction and disable constraint implementation.

Reply:
Until we have a request/need to expose this class I think we should keep it internal. Changing public interfaces is disruptive and takes a long time. With an `internal` class we give no backwards compatibility promises.